examples/voting-app/README.md

```diff
@@ -68,53 +75,175 @@ result:
   port: 5001
 ```
 
-### Add a parameters file for an environment
+## Building an App image
+
+Next, build an App image from the App definition we have created:
+
+```shell
+$ docker app build . -f voting-app.dockerapp -t voting-app:0.1.0
+[+] Building 0.6s (6/6) FINISHED
+(...) (Build output)
+sha256:46379a70af728aca32c993373b4a52655fde106953dd5a4e56aed05cde202530
+```
+
+## Inspecting an App image
+
+Now let's get detailed information about the App image we just built using the `docker app image inspect` command. Note that the `--pretty` option allows to get a human friendly output rather than the JSON default output.
```

Comment:
```suggestion
Now let's get detailed information about the App image we just built using the `docker app image inspect` command. Note that the `--pretty` option allows to get a human friendly output rather than the default JSON output.
```

Reply:
Right. Updated.